txpipe/sysmaps.py

```diff
@@ -3,17 +3,86 @@
 
 
 class TXDiagnosticMaps(PipelineStage):
+    """
+    For now, this Pipeline Stage computes a depth map using the DR1 method,
+    which takes the mean magnitude of objects close to 5-sigma S/N.
+
+    In the future we will add the calculation of other diagnostic maps
+    like airmass for use in systematics tests and covariance mode projection.
+
+    DM may in the future provide tools we can use in place of the methods
+    used here, but not on the DC2 timescale.
+
+    """
     name='TXDiagnosticMaps'
+
+    # We currently take everything from the shear catalog.
+    # In the long run this may become DM output
     inputs = [
-        ('shear_cat', MetacalCatalog),
+        ('shear_catalog', MetacalCatalog),
```

Comment:
Not sure how things are structured right now, but I wonder if one of the inputs could be a PixelScheme object, living in descformats.tx, which we will write and which contains the methods I mentioned before for a generic pixelization scheme? Or is this something that should be passed in the configuration file (e.g. a flag that says 'use this pixelization scheme or this other one')?